silk/tools/otns_manager.py

```diff
@@ -87,6 +87,16 @@ def __init__(self, server_addr: str, logger: logging.Logger):
     self.stub = visualize_grpc_pb2_grpc.VisualizeGrpcServiceStub(self.channel)
     self.logger = logger
 
+  def send_title(self, title: str):
+    """Send test title to OTNS.
+
+    Args:
+        title (str): test title.
```

Comment:
Why not also support setting position (x, y) and font_size?

Reply:
Sure thing. I wasn't sure Silk needs to care about how much of the visualization details of the title, but I can give it the same default as OTNS.